docusaurus.config.js

```diff
@@ -26,7 +26,7 @@ const config = {
         docs: {
           sidebarPath: require.resolve('./sidebars.js'),
           // Please change this to your repo.
-          editUrl: 'https://github.com/facebook/docusaurus/tree/main/packages/create-docusaurus/templates/shared/',
+          editUrl: 'https://github.com/celestiaorg/docs/blob/main/',
```

Comment:
Could also delete the line above
```suggestion
          editUrl: 'https://github.com/celestiaorg/docs/tree/main/',
```